test/cypress/e2e/oidc/oidc_auth_test.spec.js

```diff
@@ -60,33 +58,37 @@ describe('Log in via OIDC', () => {
 
     kcLogin();
 
-    cy.origin('http://localhost:5601', () => {
-      localStorage.setItem('opendistro::security::tenant::saved', '""');
-      localStorage.setItem('home:newThemeModal:show', 'false');
+    localStorage.setItem('opendistro::security::tenant::saved', '""');
+    localStorage.setItem('home:newThemeModal:show', 'false');
 
-      cy.visit('http://localhost:5601/app/dev_tools#/console');
+    cy.visit('http://localhost:5601/app/dev_tools#/console');
 
-      cy.get('a').contains('Dev Tools').should('be.visible');
+    cy.get('a').contains('Dev Tools').should('be.visible');
 
-      cy.getCookie('security_authentication').should('exist');
-    });
+    cy.getCookie('security_authentication').should('exist');
   });
 
   it('Login to Dashboard with Hash', () => {
```

Comment:
@RyanL1997 This PR updates this OIDC test that was not actually checking to see that the user resumed from the hash after login. I updated this test with a simpler hash that shows that the user is re-directed back to the getting started page. Same as the SAML test.

Reply:
Yes, That makes sense. If I remember this correctly, the current cypress didn't have the step of importing sample data, so it is totally fine to switch into this. Nice catch.